src/domain/models/bertsum_transformer.py

```diff
@@ -34,6 +47,10 @@ def forward(self, contents):
             contents["mark"],
             contents["mark_clss"],
         )
+        x = x.to(self.device)
+        segs = segs.to(self.device)
+        mask = mask.to(self.device)
```

Comment:
All those to() calls are not needed when i run the code. The DataLoader should automatically handle putting tensors on the right device. 

Also, if ever we want to go multi-GPU, this is the kind of behaviour that cannot be done when one has multiple GPUs.

Reply:
Not sure I'll use multi-GPU in CC anw.